docs/index.html

```diff
@@ -259,7 +259,7 @@ <h2>🙏 Thanks</h2>
 <p>to:</p>
 <ul>
 <li><a href="https://github.com/soasme">soasme</a> for the excellent libraries nim-markdown and nim-mustache, which provide the backbone of nimib rendering and customization</li>
-<li><a href="https://github.com/clonkk">clonkk</a> for help in a critical piece of code early on (see <a href="https://stackoverflow.com/a/64032172/4178189">this Stack Overflow answer</a>)</li>
+<li><a href="https://github.com/pietroppeter">pietroppeter</a> for help in a critical piece of code early on (see <a href="https://stackoverflow.com/a/64032172/4178189">this Stack Overflow answer</a>)</li>
```

Comment:
Are you removing yourself from the credits? 😛

Reply:
I just used sed on the hole repo lol